getting-started/repl/intergration.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Integration
+page_title: Telerik REPL for Blazor Integration
+description: Telerik REPL for Blazor integration in other Telerik Resources, Visual Studio and Visual Studio Code
+slug: blazor-repl-integration
+tags: telerik,blazor,repl,integration
+published: True
+position: 10
+---
+
+# Telerik REPL for Blazor Integration
+
+Telerik REPL for Blazor is integrated in our resources (documentation and demos), as well as Visual Studio and Visual Studio Code.
+
+In this article:
+
+* [Telerik UI for Blazor documentation Integration](#telerik-ui-for-blazor-documentation-integration)
+* [Telerik UI for Blazor demos Integration](#telerik-ui-for-blazor-demos-integration)
+* [Visual Studio and Visual Studio Code Integration](#visual-studio-and-visual-studio-code-integration)
+
+## Telerik UI for Blazor Documentation Integration
+
+The Telerik UI for Blazor documentation supports integration with the Telerik REPL for Blazor. This provides interactive experience and easy to test snippets for the various components and scenarios. Above the code snippets listed in the documentation articles you will find a toolbar with the following options:
+
+* `Edit` - click to open the source in the integrated the Telerik REPL for Blazor Editor.
+
+* `Preview` - click to run the snippet in the integrated the Telerik REPL for Blazor Result View.
+
+
+![Documentation Integration](../images/repl-docs-integration.png)
+
+
+## Telerik UI for Blazor Demos Integration
+
+The Telerik UI for Blazor demos support integration with the Telerik REPL for Blazor. In the demo window toolbar, along with the "Example" and "View Source" tabs, you will find a tab with dropdown that provides option for editing the demo in Telerik REPL for Blazor. Once you select that option, you will be redirected to the Telerik REPL for Blazor and its editor will contain the source code of the current demo, so you can directly edit and test it.
+
+![Demos Integration](../images/repl-demos-integration.png)
+
+
+## Visual Studio and Visual Studio Code Integration
+
+Telerik Extensions for [Visual Studio]({%slug getting-started-vs-integration-overview%}) and [Visual Studio Code]({%slug getting-started-vs-code-integration-overview%}) provide option to easily share your code to the Telerik REPL for Blazor with a few clicks. You can share the whole file or just a selection.
+
+The command can be used if more than 10 symbols are selected. If less symbols are selected, in Visual Studio it will appear as disabled and in Visual Studio Code you will get an error message ("Expected at least 10 characters"). In case of multiple selection every new piece will be concatenated on a new line.
```

Comment:
```suggestion
The command can be used if more than 10 symbols are selected. Otherwise Visual Studio will disable the command, and Visual Studio Code will show an error message ("Expected at least 10 characters"). In case of multiple selection every new piece will be concatenated on a new line.
```